proxy-conf-example.js

```diff
@@ -1,56 +0,0 @@
-const CLOUDSTACK_ENDPOINT = 'https://example.com';
```

Comment:
why it was removed?

Reply:
Sorry for committing this file accidentally.